packages/go/openapi/src/paths/asset-isolation.asset-group-tags.search.yaml

```
@@ -0,0 +1,74 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+post:
+  operationId: AssetGroupTagSearch
+  summary: Search Asset Group Tags
+  description: Supports searching by name on asset group tags and selectors, and by name or object ID on members.
+  tags:
+    - Asset Isolation
+    - Enterprise
+    - Community
+  requestBody:
+    description: The request body for searching asset group tags. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)